### PR TITLE
Making all intended items translatable.

### PR DESCRIPTION
**Pull request by mbehrle**
- Provide missing jinja and gettext tags.
- Provide translatable items instead of some hard coded strings in the
  code.
- Provide translatable items for some items pulled untranslated from the
  database.
- Fix some minor indentations for uniformity.

This commit is for branch 3.4, but should probably apply to current develop as well.

### CODE REVIEW

**Comment by sharoonthomas** [comment]
LGTM

File: templates/webshop/complaint.jinja [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {% extends 'base.jinja' %}
 
-{% block title %}Return #{{ complaint.reference }}{{ super() }}{% endblock %}
+{% block title %}{{ _('Return #') complaint.reference }}{{ super() }}{% endblock %}
```
Comment (tarunbhardwaj) [inline templates/webshop/complaint.jinja]: Missing closing and opening parenthesis.

File: templates/webshop/complaint.jinja [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 {% block main %}
 <div class="container">
   <div class="row">
     <div class="col-md-12">
-      <h4>Return #{{ complaint.reference }}</h4>
+      <h4>{{ _('Return #') complaint.reference }}</h4>
```
Comment (tarunbhardwaj) [inline templates/webshop/complaint.jinja]: same

**Comment by tarunbhardwaj** [comment]
LGTM

File: tryton.cfg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tryton]
-version=3.4.5.5
```
Comment (sharoonthomas) [inline tryton.cfg]: this looks like a stray change to me.

@tarunbhardwaj ?

**Comment by tarunbhardwaj** [comment]
merged manually
https://github.com/fulfilio/nereid-webshop/commit/990bb2c2d997359a07f8f0fa835229774cc34f30

**Pull request closed without merging**